develop/dev-guide-connect-to-tidb.md

```diff
@@ -10,15 +10,29 @@ aliases: ['/zh/tidb/dev/connect-to-tidb']
 
 TiDB 支持 [MySQL 客户端/服务器协议](https://dev.mysql.com/doc/internals/en/client-server-protocol.html)。这使得大多数客户端驱动程序和 ORM 框架可以像连接到 MySQL 一样地连接到 TiDB。
 
+你可以选择使用 MySQL Client 或 MySQL Shell 连接到 TiDB。
+
 ## MySQL Client
 
-你可以使用 MySQL Client 作为 TiDB 的命令行工具。在 [MySQL Shell 官方文档](https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html) 你可以找到不同操作系统的安装方式。在安装完后你可以使用如下命令行连接到 TiDB：
+你可以使用 MySQL Client 作为 TiDB 的命令行工具连接到 TiDB。下面以基于 YUM 的 Linux 发行版为例，介绍如何按照 MySQL Client。
```

Comment:
Did we need to add the `<SimpleTabs>` tag to give some other platform examples to the audience?

Reply:
Done https://github.com/pingcap/docs-cn/pull/12987/commits/ac60843553b4a5c773884ef8d14b4ec7c27913ed

Preview:

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/60599231/218444706-3765bdcf-aeda-49bc-b06d-ffd8cb002e19.png">